README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
At least in the United States, freedom of speech/expression doesn't apply on private property, even when it's publically accessible. Case in point: I can't protest inside my local shopping mall because it's private property, but anybody can still walk in.

GitHub is a shopping mall. It's not a government owned plot of land, it's a place with open doors and a set of rules. Don't like the rules? Make your own repo.